assets/src/dashboard/app/views/myStories/content/index.js

```diff
@@ -0,0 +1,172 @@
+  Layout,
+  StandardViewContentGutter,
+  DefaultParagraph1,
+} from '../../../../components';
+import {
+  VIEW_STYLE,
+  STORY_ITEM_CENTER_ACTION_LABELS,
+} from '../../../../constants';
+import {
+  UsersPropType,
+  TagsPropType,
+  CategoriesPropType,
+  StoriesPropType,
+  StoryActionsPropType,
+} from '../../../../types';
+import {
+  FilterPropTypes,
+  ViewPropTypes,
+  PagePropTypes,
+  SortPropTypes,
+} from '../../../../utils/useStoryView';
+import FontProvider from '../../../font/fontProvider';
+import { NoResults, StoryGridView, StoryListView } from '../../shared';
+
+function Content({
+  allPagesFetched,
+  categories,
+  filter,
+  isLoading,
+  page,
+  search,
+  sort,
+  stories,
+  storyActions,
+  tags,
+  users,
+  view,
+}) {
+  const contentView = useMemo(() => {
+    switch (view.style) {
+      case VIEW_STYLE.GRID:
+        return (
+          <StoryGridView
+            trashStory={storyActions.trashStory}
+            updateStory={storyActions.updateStory}
+            createTemplateFromStory={storyActions.createTemplateFromStory}
+            duplicateStory={storyActions.duplicateStory}
+            stories={stories}
+            users={users}
+            centerActionLabelByStatus={STORY_ITEM_CENTER_ACTION_LABELS}
+            bottomActionLabel={__('Open in editor', 'web-stories')}
+          />
+        );
+      case VIEW_STYLE.LIST:
+        return (
+          <StoryListView
+            stories={stories}
+            storySort={sort.value}
+            storyStatus={filter.value}
+            sortDirection={sort.direction}
+            handleSortChange={sort.set}
+            handleSortDirectionChange={sort.setDirection}
+            tags={tags}
+            categories={categories}
+            users={users}
+          />
+        );
+      default:
+        return null;
+    }
+  }, [
+    categories,
+    filter.value,
+    sort,
+    storyActions,
+    stories,
+    tags,
+    users,
+    view.style,
+  ]);
+
+  const BodyContent = useMemo(() => {
+    if (stories.length > 0) {
+      return (
+        <StandardViewContentGutter>
+          {contentView}
+          <InfiniteScroller
+            canLoadMore={!allPagesFetched}
+            isLoading={isLoading}
+            allDataLoadedMessage={__('No more stories', 'web-stories')}
+            onLoadMore={page.requestNextPage}
+          />
+        </StandardViewContentGutter>
+      );
+    } else if (search.keyword.length > 0) {
+      return <NoResults typeaheadValue={search.keyword} />;
+    }
+
+    return (
+      <DefaultParagraph1>
+        {__('Create a story to get started!', 'web-stories')}
+      </DefaultParagraph1>
+    );
+  }, [
+    stories.length,
+    isLoading,
+    allPagesFetched,
+    page.requestNextPage,
+    search.keyword,
+    contentView,
+  ]);
+
+  return (
+    <Layout.Scrollable>
+      <FontProvider>
+        <TransformProvider>
+          <UnitsProvider pageSize={view.pageSize}>{BodyContent}</UnitsProvider>
```

Comment:
On this page, let's flatten Body Content into the main render since we already are calling this component `Content`. Plus we have `contentView`. It might be worth breaking out `contentView` into a `StoryContent` component. Currently this file has 3 branches. Stories - List, Stories - Grid, No Stories.

Reply:
Sounds great! 
What I went with was pulling it apart into `StoriesView` (where the grid/list live) and `EmptyView` where the default and no search results live to better group the logic. 

I added new test files into content/test for each component and I opted to just add stories for the two new components in with content/stories. This might be because I feel like content/viewStories/stories right next to content/stories feels... confusing or awkward?